superset-frontend/src/components/Menu/Menu.tsx

```diff
@@ -56,32 +56,52 @@ export interface MenuProps {
 }
 
 const StyledHeader = styled.header`
+  & > nav {
+    border: none;
+  }
   .navbar-brand {
     display: flex;
     flex-direction: column;
     justify-content: center;
   }
 
+  .nav > li > a {
+    padding: ${({ theme }) => theme.gridUnit * 4}px;
+  }
+
   .navbar-nav > li > a {
+    border-bottom: none;
+    &:focus {
+      border-bottom: none;
+    }
     &:after {
       content: '';
       position: absolute;
       bottom: -3px;
-      left: 0;
-      width: 100%;
+      left: 50%;
+      width: 0;
       height: 3px;
       background-color: ${({ theme }) => theme.colors.primary.base};
       opacity: 0;
-      transition: opacity ${({ theme }) => theme.transitionTiming * 2}s;
+      transform: translateX(-50%);
+      transition: all ${({ theme }) => theme.transitionTiming}s !important;
```

Comment:
yikes... why do we need an `!important` here?

Reply:
Yeahhhhhhh, I have a hatred for `!important` too. This was just because it's overriding some deeeep selector stuff from Bootstrap. I can add the full selector stack instead, I suppose. Will look into it.

Reply:
I think it's worth seeing how messy it looks either way. It might be better to show the full selector stack regardless, since then it's more obvious to developers that this is fragile logic. `!important` can hide a bunch of nuance